modules/TwintPool.py

```diff
@@ -147,14 +160,15 @@ def row_tweet_to_urls(row):
         neo4j_df['hashtags'] = df['hashtags'].apply(lambda x: [{'text': ht} for ht in x])
         neo4j_df['user_followers_count'] = None
         neo4j_df['user_friends_count'] = None
+        neo4j_df['date'] = df['date']
         # neo4j_df['user_created_at'] = None
```

Comment:
@webcoderz  afaict, and following the naming convention, `neo4j_df['user_created_at']` is for user account creation date and `neo4j_df['created_at']` for tweet creation date. So instead of adding `neo4j['date']`, we should put it in `neo4j['user_created_at']` or `neo4j['created_at']`

Reply:
yea we don't really need it i made this commit before i noticed the bug with  neo4jdf["created_at"]  theyre both the same now .. in twint when you get the user info the user created date would be df["join_datetime"]